openapi.yaml

```diff
@@ -322,6 +322,36 @@ info:
         }'
     ```
 
+    # Rate Limiting
+
+    With the Linode API, you can make up to 1600 general API requests every two minutes per user as
+    determined by IP adddress or by OAuth token.
+
+    **Note:** There may be rate limiting applied at other levels outside of the API, for example, at the load balancer.
+
+    `/stats` endpoints have their own dedicated limits of 100 requests per minute per user.
+    These endpoints are:
+
+    * [View Linode Statistics](https://api.linode.com/v4/linode/instances/{linodeId}/stats)
+    * [View Linode Statistics (year/month)](https://api.linode.com/v4/linode/instances/{linodeId}/stats/{year}/{month})
+    * [View NodeBalancer Statistics](https://api.linode.com/v4/nodebalancers/{nodeBalancerId}/stats)
+    * [List Managed Stats](https://api.linode.com/v4/managed/stats)
+
```

Comment:
support tickets POST also has a limit: https://bits.linode.com/LinodeAPI/apinext/blob/development/apinext/blueprints/support/tickets.py#L136

Reply:
Thank you - I added a section for support tickets